### PR TITLE
kusion: add version v0.8.0

### PR DESCRIPTION
**Pull request by howieyuen**
Close https://github.com/ScoopInstaller/Main/issues/4333
Add [kusion](https://github.com/KusionStack/kusion) to Main
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Hi, this project is pretty new and does not satisfy the Main criteria yet. Please create an issue for it instead.

**Comment by howieyuen** [comment]
> Hi, this project is pretty new and does not satisfy the Main criteria yet. Please create an issue for it instead.

Thanks, I'd do it now~

**Comment by rashil2000** [comment]
You're welcome :)

**Comment by howieyuen** [comment]
kusion has 500 stars now, can you reopen this PR?

**Comment by rashil2000** [comment]
Sure, please update it to the latest version

**Comment by howieyuen** [comment]
I have updated the manifest to the last version

**Comment by rashil2000** [comment]
- Please follow consistency in spacing - we use a tab width of 4 spaces everywhere.
- Shortcuts are not required for CLI tools
- Move `bin` outside of architecture field

**Comment by howieyuen** [comment]
> * Please follow consistency in spacing - we use a tab width of 4 spaces everywhere.
> * Shortcuts are not required for CLI tools
> * Move `bin` outside of architecture field

thx, updated.

File: bucket/kusion.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "0.8.0",
+    "description": "Platform Engineering Engine: Deliver intentions to Kubernetes and Clouds",
+    "homepage": "http://github.com/KusionStack/kusion",
+    "license": "Apache License 2.0",
```
Comment (rashil2000) [inline bucket/kusion.json:5]:
```suggestion
    "homepage": "https://kusionstack.io",
    "license": "Apache-2.0",
```

**Pull request merged**